### DISCUSSION TITLE
Austausch Bachelorpraktikum, Projektseminar

### DISCUSSION THREAD

**Discussion by HendrikSmd** [lectureStudio/lectureStudio#419] [General]
Hallo, der Thread hier ist für den Austausch zwischen Bachelorpraktikum und Projektseminar gedacht. Der Austausch soll helfen unseren Code ineinander zu überführen.

**Comment by sebastiandeppisch** [comment]
Welchen Branch für lectureStreaming/lectureStudio muss ich auschecken, um einen aktuellen und funktionierenden Stand des Messengers zu haben?

**Reply by HendrikSmd** [reply]
Sorry, für die späte Nachricht. Sowohl lectureStudio als auch lectureStreaming sind jetzt auf dev/dedicated aktuell.
